Core/Graphics/Source/GraphicsImpl.cpp

```diff
@@ -30,7 +31,7 @@ namespace Babylon
 
         auto& init = m_state.Bgfx.InitState;
         init.type = s_bgfxRenderType;
-        init.resolution.reset = BGFX_RESET_VSYNC | BGFX_RESET_MSAA_X4 | BGFX_RESET_MAXANISOTROPY;
```

Comment:
I'm not sure we should split this into parts. The intention is that they are the same. Can you think of a reason why we want the reset flags to be different on purpose? IMO we can just use #ifdefs here and put a comment saying it's broken for Android/iOS like you did but in one place.

Reply:
Using #ifdef was my first choice but you proposed to put the flags in header, per platform.

Reply:
Sorry, yes, I did, but I wasn't thinking about what the intention was.

Reply:
Let me take care of this since I broke the Android version.